Modules/System/Data Compression/src/DataCompressionImpl.Codeunit.al

```diff
@@ -108,6 +108,18 @@ codeunit 421 "Data Compression Impl."
         end;
     end;
 
+    [TryFunction]
+    procedure IsZip(InputInStream: InStream)
```

Comment:
Shouldn't it be possible to use a similiar pattern as for the GZIP Compressed Stream.
See Header: https://en.wikipedia.org/wiki/ZIP_(file_format)#File_headers

Reply:
Yeah. I thought about that too. Delegating the check to the actual routines that are already in place seemed better to me rather than reinvent the wheel.
If the header check is preferred, I can do that.

Reply:
We can wait for Feedback from microsoft on this topic.

Reply:
Done.